org.sugarlabs.Pippy.json

```diff
@@ -0,0 +1,282 @@
+{
+    "app-id": "org.sugarlabs.Pippy",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "separate-locales": false,
+    "command": "sugarapp",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--device=dri",
+        "--device=all",
+        "--env=SUGAR_BUNDLE_ID=org.sugarlabs.Pippy",
+        "--env=SUGAR_BUNDLE_PATH=/app/share/sugar/activities/Pippy.activity"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/info",
+        "/share/man"
+    ],
+    "modules": [
+        "shared-modules/dbus-glib/dbus-glib-0.110.json",
+        "shared-modules/intltool/intltool-0.51.json",
+        "shared-modules/pygame/pygame-1.9.4.json",
+        {
+            "name": "empy",
+            "cleanup": [ "*" ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} empy"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/3b/95/88ed47cb7da88569a78b7d6fb9420298df7e99997810c844a924d96d3c08/empy-3.3.4.tar.gz",
+                    "sha256": "73ac49785b601479df4ea18a7c79bc1304a8a7c34c02b9472cf1206ae88f01b3"
+                }
+            ],
+            "post-install": [
+                "chmod +x /app/lib/python3.7/site-packages/em.py",
+                "mkdir -p /app/bin/",
+                "ln -s /app/lib/python3.7/site-packages/em.py /app/bin/empy"
+            ]
+        },
+        {
+            "name": "perl-XML-Simple",
+            "cleanup": [ "*" ],
+            "no-autogen": true,
+            "build-options": {
+                "no-debuginfo": true
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-Simple-2.25.tar.gz",
+                    "sha256": "531fddaebea2416743eb5c4fdfab028f502123d9a220405a4100e68fc480dbf8"
+                },
+                {
+                    "type": "file",
+                    "path": "perl-MakefilePL-Makefile",
```

Comment:
Might as well just use the `simple` build system type IMO. Not a big deal though.

Reply:
You are right, it's one less file.

After this app, I am planning to do a mass update/cleanup of all of the sugarlabs apps (e.g move most of these modules to a BaseApp). Ill make sure include this, then.